partner_contact_in_several_companies/models/res_partner.py

```diff
@@ -12,8 +12,7 @@ class ResPartner(models.Model):
         [('standalone', _('Standalone Contact')),
          ('attached', _('Attached to existing Contact')),
          ],
-        compute='_get_contact_type',
-        required=True, select=1, store=True,
+        required=True,
```

Comment:
With compute, store and select I have this error.

So for this with WIP migration I decide to have onchange. Will go further to see if it is possible to keep this.

Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/__init__.py", line 859, in emit
    msg = self.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 732, in format
    return fmt.format(record)
  File "/opt/apps/0k-oe/odoo/odoo/netsvc.py", line 108, in format
    return DBFormatter.format(self, record)
  File "/opt/apps/0k-oe/odoo/odoo/netsvc.py", line 102, in format
    return logging.Formatter.format(self, record)
  File "/usr/lib/python2.7/logging/__init__.py", line 471, in format
    record.message = record.getMessage()
  File "/usr/lib/python2.7/logging/__init__.py", line 335, in getMessage
    msg = msg % self.args
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 388, in __str__
    return "%s.%s" % (self.model_name, self.name)
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
....
...
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
  File "/opt/apps/0k-oe/odoo/odoo/fields.py", line 357, in __getattr__
    return self._attrs[name]
RuntimeError: maximum recursion depth exceeded
Logged from file fields.py, line 463